docs/en/Getting-Started-on-Cluster.md

```diff
@@ -0,0 +1,348 @@
+
+## Download Alluxio
+
+First, [download the Alluxio release](http://www.alluxio.org/download) locally. You can
+download the latest {{site.ALLUXIO_RELEASED_VERSION}} release pre-built for various versions of
+Hadoop from the [Alluxio download page](http://www.alluxio.org/download).
+
+Second, `scp` the release tarball to all the Linux nodes. Note that, feel free to use scriptable way (such as Ansible)
+to distribute the tarball and automate the following process on multiple machines. In this documentation,
+manually commands are shown and most of them should be executed on all the nodes, to help user better understand
+how Alluxio master and Alluxio workers start individually and how they work together.
+
+Next, you can unpack the download with the following commands on all the nodes. Your file name may be different
+depending on which pre-built binaries you have downloaded.
+
+```bash
+$ tar -xzf alluxio-{{site.ALLUXIO_RELEASED_VERSION}}-bin.tar.gz
+$ cd alluxio-{{site.ALLUXIO_RELEASED_VERSION}}
+```
+
+This will create a directory `alluxio-{{site.ALLUXIO_RELEASED_VERSION}}` with all of the Alluxio
+source files and Java binaries.
+
+## Configure Alluxio
+
+Before we start Alluxio, we have to configure it. Please ensure all the subsequent commands are run on all the Linux nodes.
+
+In the `${ALLUXIO_HOME}/conf` directory, create the `conf/alluxio-site.properties` configuration
+file from the template.
+
+```bash
+$ cp conf/alluxio-site.properties.template conf/alluxio-site.properties
+```
+
+Update `alluxio.master.hostname` in `conf/alluxio-site.properties` to the hostname of the machine
+you plan to run Alluxio Master on. Let's name it Alluxio Master node, and assume its hostname is
+**ALLUXIO_MASTER_HOSTNAME**.
+
+```bash
+$ echo "alluxio.master.hostname=ALLUXIO_MASTER_HOSTNAME" >> conf/alluxio-site.properties
+```
+
+### Configure for AWS
+
+Please ensure all the subsequent commands are run on all the Linux nodes.
+
+If you have an Amazon AWS account with your access key id and secret key, you can update your
+Alluxio configuration now in preparation for interacting with Amazon S3 later in this guide. Add
+your AWS access information to the Alluxio configuration by adding the keys to the
+`conf/alluxio-site.properties` file. The following commands will update the configuration.
+
+```bash
+$ echo "aws.accessKeyId=AWS_ACCESS_KEY_ID" >> conf/alluxio-site.properties
+$ echo "aws.secretKey=AWS_SECRET_ACCESS_KEY" >> conf/alluxio-site.properties
+```
+
+You will have to replace **AWS_ACCESS_KEY_ID** with your AWS access key id, and
+**AWS_SECRET_ACCESS_KEY** with your AWS secret access key. Now, Alluxio is fully configured for the
+rest of this guide.
+
+## Validate Alluxio environment
+
+Before starting Alluxio, you might want to make sure that your system environment is ready for running
+Alluxio services. You can run the following command to validate environment on all nodes:
+
+```bash
+$ ./bin/alluxio validateEnv all
```

Comment:
This requires `conf/workers` to be set up, but the guide doesn't assume that

Reply:
ping

Reply:
done